src/mpyl/steps/deploy/k8s/__init__.py

```diff
@@ -90,6 +109,15 @@ def deploy_helm_chart(
         delete_existing,
     )
 
+    if output.success and run_properties.target:
```

Comment:
I assume you meant to check for target `== Target.PRODUCTION`?

Reply:
ah yes